modules/hal_bouffalolab/include/bflb_hbn.h

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2021-2025 Gerson Fernando Budke <nandojve@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_HAL_BFLB_HBN_H_
+#define ZEPHYR_HAL_BFLB_HBN_H_
+
+#ifdef CONFIG_SOC_SERIES_BL60X
+	#include <bl602_hbn.h>
```

Comment:
we don't use indents for #if followed by #include in any other socs, remove indent

Reply:
fixed